### PR TITLE
Spat kafka stream

### PR DESCRIPTION
**Pull request by animeshb27**
# PR Details
## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by arayama — COMMENTED** [review]
Looks okay to me.

File: Dockerfile [hunk 1]
```diff
@@ -33,6 +32,12 @@ RUN cmake .
 RUN make 
 RUN make install
 
+WORKDIR /home/V2X-Hub/ext/date
```
Comment (dan-du-car) [inline Dockerfile:35]: Where is this date folder from?

**Comment by dan-du-car** [comment]
Looking good! Thanks! I left one comment.

**Comment by dan-du-car** [comment]
It looks like the CI failed. Could you also please address this CI failure?

**Review by dan-du-car — APPROVED** [review]

**Comment by paulbourelly999** [comment]
Already included in #194

**Pull request closed without merging**